CHANGES.md

```diff
@@ -48,6 +54,7 @@ To be released.
  -  `Swarm<T>` became not to receive states from trusted peers.
     [[#1061], [#1102]]
  -  `Swarm<T>` became not to retry when block downloading.  [[#1062], [#1102]]
+ -  `Swarm<T>` became not to fill states from trusted peers.  [[#1117]]
```

Comment:
```suggestion
 -  `Swarm<T>` became not to fill states from trusted peers, because now states can be
    validated rather than trusted due to MPT.  [[#1117]]
```

Reply:
Amended it in 1636849.